ext/random/php_random_module.h

```diff
@@ -0,0 +1,67 @@
+/*
```

Comment:
The file name is wrong and will cause build issues, I think. This should be named `php_random.h` instead. See build/print_include.awk

```
../php-src/ext/tokenizer/php_tokenizer.h
21:#define phpext_tokenizer_ptr &tokenizer_module_entry

../php-src/ext/xsl/php_xsl.h
21:#define phpext_xsl_ptr &xsl_module_entry
```

Reply:
If changed to php_random.h, the include guard conflicts with php_random.h in ext/standard.
Use PHP_RANDOM_MODULE_H for the include guard only.